### PR TITLE
ci: fix up make checks 

### PR DESCRIPTION
**Pull request by acramsay**

### CODE REVIEW

**Review by acramsay — COMMENTED** [review]
Let's review and merge this after the holidays when everyone is back.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -33,7 +36,7 @@ jobs:
 
             - name: Check Code Generation
               run: |
-                  make generate
+                  make gen-all
```
Comment (adrielp) [inline .github/workflows/build.yml:39]: Let's keep this as `generate` for consistency with the upstream.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -25,6 +25,9 @@ jobs:
                   go-version: ${{ matrix.go }}
                   cache: false
 
+            - name: Make prep
```
Comment (adrielp) [inline .github/workflows/build.yml:28]: I don't see `prep` added as a rule in the makefile within this diff.

File: Makefile [hunk 3]
```diff
@@ -76,9 +76,14 @@ for-all:
 lint-all:
 	$(MAKE) for-all DIRS="$(PKG_DIRS)" CMD="$(MAKE) lint"
 
+# Needeed temporarily in order to run ci jobs successfully
```
Comment (adrielp) [inline Makefile:79]: Not following these changes that well. What's the goal with these changes? It wouldn't surprise me here if the `checks` rule didn't actually work here. But IIRC it wasn't used in the workflows & needed some additional TLC to make contribution ready.

File: .github/workflows/build.yml (hunk 2 above)
Comment (acramsay) [reply .github/workflows/build.yml:28]: Prep already existed and there was no change to it. I.e. it's exists but not in the diff

File: .github/workflows/build.yml (hunk 1 above)
Comment (acramsay) [reply .github/workflows/build.yml:39]: That's a good point. The change was motivated by the naming conventions within our makefile. We can definitely change it back to stay in line with the contrib repo.

What do you think about about keeping both?
- The `generate` command has the logic
- The `gen-all` command is an alias for `generate`
- The github workflow runs `make generate`

**Review by adrielp — APPROVED** [review]

File: Makefile (hunk 3 above)
Comment (acramsay) [reply Makefile]: Addressed in live code review and reverted this change.

**Pull request merged**